app/models/box.rb

```diff
@@ -34,6 +34,8 @@ def build_samples(batch, concentration_exponents:, replicates:)
           concentration_number: 1,
           concentration_exponent: exponent,
           replicate: replicate,
+          institution: institution,
+          site: site,
```

Comment:
It's not enough to rely on `Batch#build_sample` :facepalm:

Reply:
Yeah, it would only set the institution, but not the site. And technically, there is no inherent guarantee that the institutions of batch and box match, that's why I'm setting the institution explicitly to the box's here.